### PR TITLE
Fix nullable values in Coingecko Market type

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/356?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#356** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/356?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#355** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/355?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#354** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/354?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#353** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/353?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @youngkidwarrior and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/utils/coin-gecko/index.tsx [hunk 1]
```diff
@@ -28,11 +28,13 @@ export const MarketDataSchema = z
...
-      currency: z.string(),
-      percentage: z.number(),
-    }),
+    roi: z
+      .object({
+        times: z.number(),
+        currency: z.string(),
+        percentage: z.number(),
+      })
+      .nullable(),
```
Comment (0xBigBoss) [inline packages/app/utils/coin-gecko/index.tsx:37]: interesting. what is roi anyways? 

thx for scheming out the whole object. next we should only parse and validate the keys we care about.

Comment (youngkidwarrior) [reply packages/app/utils/coin-gecko/index.tsx:37]: Totally

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Apr 26, 6:54 PM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/356).

**Pull request merged**